### PR TITLE
Make grammar changes on Code of Conduct page

### PR DESCRIPTION
**Pull request by redoomed1**
Changes proposed in this PR:

- Correct grammatical errors on the Code of Conduct page
- Update aliasing link that I overlooked in https://github.com/privacyguides/privacyguides.org/pull/2480 :sweat_smile: 

- [x] I have disclosed any relevant conflicts of interest in my post.
- [x] I agree to grant Privacy Guides a perpetual, worldwide, non-exclusive, transferable, royalty-free, irrevocable license with the right to sublicense such rights through multiple tiers of sublicensees, to reproduce, modify, display, perform, relicense, and distribute my contribution as part of this project.
- [x] I am the sole author of this work. 
- [x] I agree to the [Community Code of Conduct](https://www.privacyguides.org/coc).

### CODE REVIEW

File: docs/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ The following behaviors are considered harassment and are unacceptable within ou
 
 - Any other conduct which would reasonably be considered inappropriate in a professional setting.
 - Public and/or private harassment of any kind.
-- Publishing others' private information, such as a physical address and/or email address, without a persons explicit permission.
+- Publishing others' private information, such as a physical address and/or an email address, without a person's explicit permission.
```
Comment (kimg45) [inline docs/CODE_OF_CONDUCT.md:37]:
```suggestion
- Publishing others' private information, such as a physical address and/or an email address, without their explicit permission.
```

File: docs/CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -12,11 +12,11 @@ What we expect from members of our communities:
...
 
 2. **Do not abuse our willingness to help**
 
-      Our community members are not free tech support. We are happy to help with specific steps for individual's, privacy journey, if they are willing to put in effort. We are not obligated to answer endless, repetitive questions, about general computer problems solvable with a simple internet search. **Do not** become a [help vampire](https://slash7.com/2006/12/22/vampires).
+      Our community members are not free tech support. We are happy to help with specific steps for an individual's privacy journey, if they are willing to put in effort. We are not obligated to answer endless, repetitive questions about general computer problems solvable with a simple internet search. **Do not** become a [help vampire](https://slash7.com/2006/12/22/vampires).
```
Comment (kimg45) [inline docs/CODE_OF_CONDUCT.md:19]:
```suggestion
      Our community members are not free tech support. We are happy to help with specific steps for your privacy journey, if you are willing to put in effort. We are not obligated to answer endless, repetitive questions about general computer problems solvable with a simple internet search. **Do not** become a [help vampire](https://slash7.com/2006/12/22/vampires).
```

**Review by kimg45 — CHANGES_REQUESTED** [review]
A few wording changes I think would make it read better.

**Comment by dngray** [comment]
Agreed i do think these rewords are better LGTM.

**Review by dngray — APPROVED** [review]

**Review by dngray — APPROVED** [review]

**Pull request merged**